configs/common/packages.yaml

```diff
@@ -265,9 +265,11 @@ packages:
   py-urllib3:
     require: '@1.26.12'
   qt:
-    require: '@5'
+    require:
+    - '@5'
   scotch:
-    require: '@7.0.4 +mpi+metis~shared~threads~mpi_thread+noarch+esmumps'
+    require:
+    - '@7.0.4 +mpi+metis~shared~threads~mpi_thread+noarch+esmumps'
```

Comment:
Does this mean we are using scotch 7.0.4 or are we moving to 7.0.7? (sorry this is my lack of spack-stack knowledge question)

Reply:
This is still scotch 7.0.4. If it works, then there is no need to update it. The purpose of spack-stack release 1.9.2 is to provide a bug fix / hot fix for the existing 1.9.0 and 1.9.1 releases. Bug fix releases only contain the necessary changes to fix bugs, no changes that aren't necessary.

Reply:
Do we know if scotch 7.0.4 works on other machines? Or just Ursa... I'm assuming it's going to work on other machines but don't know. Either way WW3 is ready to handle a mix of the two if necessary.